<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="processedLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>1291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start WebCam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Pause/Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="trackButton">
      <property name="text">
       <string>Track Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>590</y>
     <width>311</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="hmaxLabel">
        <property name="text">
         <string>H_max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hmaxSlider">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="smaxLabel">
        <property name="text">
         <string>S_max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="smaxSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="vmaxLabel">
        <property name="text">
         <string>V_max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="vmaxSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>281</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="hminLabel">
        <property name="text">
         <string>H_min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hminSlider">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="sminLabel">
        <property name="text">
         <string>S_min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sminSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="vminLabel">
        <property name="text">
         <string>V_min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="vminSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>239</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCheckBox" name="color1Check">
        <property name="text">
         <string>Track color 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="color1Button">
        <property name="text">
         <string>Set Color 1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QCheckBox" name="color2Check">
        <property name="text">
         <string>Track color 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="color2Button">
        <property name="text">
         <string>Set Color 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QCheckBox" name="color3Check">
        <property name="text">
         <string>Track color 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="color3Button">
        <property name="text">
         <string>Set Color 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>590</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="color1Label">
      <property name="text">
       <string>HSV Value Range:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="color2Label">
      <property name="text">
       <string>HSV Value Range:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="color3Label">
      <property name="text">
       <string>HSV Value Range:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
